src/actions/index.js

```diff
@@ -11,8 +11,10 @@ export const DECREMENT = 'DECREMENT';
 
 export const increment = () => {
   // Fill in this function
+  return {type: INCREMENT}
```

Comment:
Well done here. You recognized that you didn't need to pass a 'payload' through, so you didn't.